pkg/route/api/validation/validation_test.go

```diff
@@ -247,6 +247,19 @@ func TestValidateTLS(t *testing.T) {
 			},
 			expectedErrors: 1,
 		},
+		{
+			name: "Double escaped newlines",
+			route: &api.Route{
+				TLS: &api.TLSConfig{
+					Termination:              api.TLSTerminationReencrypt,
+					Certificate:              "d\\nef",
```

Comment:
leave these as-is (don't use raw strings), since the interpreted value is how a double-escaped newline in JSON will appear after being parsed ("\" followed by a newline)